plugins/modules/aci_cloud_external_epg.py

```diff
@@ -0,0 +1,294 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Anvitha Jain (@anvitha-jain) <anvjain@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: aci_cloud_external_epg
+short_description: Manage Cloud External EPG (cloud:ExtEPg)
+description:
+- Configures WAN router connectivity to the cloud infrastructure.
+notes:
+- More information about the internal APIC class B(cloud:ExtEPg) from
+  L(the APIC Management Information Model reference,https://developer.cisco.com/docs/apic-mim-ref/).
```

Comment:
I think we can move the notes section to the bottom of the description section, if applicable